src/connection.h

```diff
@@ -144,7 +144,8 @@ struct connListener {
     int bindaddr_count;
     int port;
     ConnectionType *ct;
-    void *priv; /* used by connection type specified data */
+    void *priv1; /* used by connection type specified data */
+    void *priv2; /* used by connection type specified data */
```

Comment:
IMO, I do not like name the variable name as priv1, priv2.  Later, we will not know the meaning they represent.  I believe priv1 represents unixsocketperm, and thus maybe void *group or ugroup is better than now to represent unixsocketgroup

Reply:
see https://github.com/valkey-io/valkey/pull/901#pullrequestreview-2237090821